src/include/nanobench.h

```diff
@@ -99,6 +99,12 @@
 #    endif
 #endif
 
+#if defined(__riscv) && !defined(ANKERL_NANOBENCH_DISABLE_PERF_COUNTERS)
```

Comment:
We want to control RISCV_PERF_COUNTERS separately, as in my version. We might want to replace Linux perf calls even for Linux platform, e.g. Linux 4.19 doesn't have support for RISC-V perf